src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+import React, {Component} from 'react';
+
+function ErrorBoundary(WrappedComponent) {
+    return class extends Component{
+        constructor(props, context) {
+            super(props, context);
+
+            this.state = {
+                error: null,
+            };
+        }
+
+        componentDidCatch(error) {
+            this.setState({ error });
+        }
+
+        render() {
+            console.log(this.state);
+            if (this.state.error) {
+                alert('Error, try to whitelist the site in AdBlocker/Cookie Whitelist');
```

Comment:
I don't think this is the right way to warn people about this. I think a `console.error` log would be reasonable though. Maybe something like:

> If you're using an ad-blocker or have disabled cookies,
> Consider adding this website to the whitelist.

We should also keep in mind that not _all_ errors are caused by this so our wording shouldn't be overly assertive that this is definitely the cause/fix. Users might see an error even if they aren't using adblockers and haven't disabled cookies, and that would be confusing.